lib/experimental/fonts/font-library/class-wp-font-family.php

```diff
@@ -304,14 +304,41 @@ private function sanitize() {
 				),
 			),
 		);
+
 		// Creates a new WP_Theme_JSON object with the new fonts to
 		// leverage sanitization and validation.
-		$theme_json     = new WP_Theme_JSON_Gutenberg( $fonts_json );
-		$theme_data     = $theme_json->get_data();
+		$theme_json = new WP_Theme_JSON_Gutenberg( $fonts_json );
+		$theme_data = $theme_json->get_data();
+
 		$sanitized_font = ! empty( $theme_data['settings']['typography']['fontFamilies'] )
 			? $theme_data['settings']['typography']['fontFamilies'][0]
 			: array();
-		$this->data     = $sanitized_font;
+
+		// Ensure slugs are kebab-cased and font families are wrapped in quotes.
+		if ( isset( $sanitized_font['slug'] ) ) {
```

Comment:
Could this be a method?

Reply:
@matiasbenedetto I moved this formatting functionality to utils, what do you think? https://github.com/WordPress/gutenberg/pull/55393/commits/efa76025c2c7d8a42134032923b24713702bbde5

Reply:
Tha's better. I reduced the function's scope to only one thing instead of 2 and renamed it accordingly: https://github.com/WordPress/gutenberg/pull/55393/commits/a414e3f40391793fb1372a769109875e561bf926

Reply:
We would need to add PHP unit tests to the function.